Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewModel.swift

```diff
@@ -220,4 +274,25 @@ struct CreditCardBottomSheetViewModel {
         let decryptedCardNum = autofill.decryptCreditCardNumber(encryptedCCNum: card.ccNumberEnc)
         return decryptedCardNum ?? ""
     }
+
+    func listStoredCreditCards(_ completionHandler: @escaping ([CreditCard]?) -> Void) {
```

Comment:
💼 TODO: Maybe add test
Note: This does use rust autofill which doesn't work well with BR

Reply:
I would even skip this as this particular type is tested as a part of autofill 
@razvanlitianu I will leave it upto you